frontend/src/widgets/RecipientsWithNoTtaWidget.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { RECIPIENTS_WITH_NO_TTA_PER_PAGE } from '../Constants';
+import HorizontalTableWidget from './HorizontalTableWidget';
+import WidgetContainer from '../components/WidgetContainer';
+import useWidgetPaging from '../hooks/useWidgetPaging';
+
+function RecipientsWithNoTtaWidget({
+  data,
+  loading,
+  resetPagination,
+  setResetPagination,
+  perPageNumber,
```

Comment:
why is this a prop and also a constant on line 3? might be simpler just to use the constant since it can't change

Reply:
updated.